### ISSUE TITLE
About the BatchNorm

### ISSUE THREAD

**Issue by Angzz** [xingyizhou/CenterNet#56]
Hi, for the hourglass network, does the BN layers in the backbone need to be fixed during training?

**Comment by xingyizhou** [comment]
Our model is trained by disabling cudnn BN (but still using pytorch cuda BN). I haven't tested other settings.

**Comment by Angzz** [comment]
Thanks, another question, does the param `si` affect the results to much? I am trying to reproduce your results but still far behind.
